params.yml

```diff
@@ -52,25 +52,25 @@ args:
 
   # Docker images and repositories used to spin up services.
   # zkevm_prover_image: hermeznetwork/zkevm-prover:v8.0.0-RC5-fork.12
-  zkevm_prover_image: hermeznetwork/zkevm-prover:v6.0.3-RC20
+  zkevm_prover_image: hermeznetwork/zkevm-prover:v8.0.0-RC5-fork.12
   # zkevm_prover_image: hermeznetwork/zkevm-prover:v4.0.19
 
   zkevm_node_image: hermeznetwork/zkevm-node:v0.7.0
   cdk_validium_node_image: 0xpolygon/cdk-validium-node:0.7.0-cdk
-  cdk_node_image: ghcr.io/0xpolygon/cdk:0.0.9
+  cdk_node_image: arnaubennassar/cdk:use-l1info-7
```

Comment:
nit: just a reminder to make sure we use the official images before merging this PR - I know you mentioned it on Slack :)